packages/graphql/src/translate/cypher-builder/clauses/Clause.ts

```diff
@@ -40,3 +40,17 @@ export abstract class Clause extends CypherASTNode {
         return new CypherEnvironment(prefix);
     }
 }
+
+/** For testing purposes only */
+export class TestClause extends Clause {
```

Comment:
Do we want this in our "production code"?

Reply:
mmm, the only alternative I can think of is creating a new "tests" folder in cypher-builder, but it will only contain these utils classes :confused:

Reply:
I ended up moving it to a separate file inside `utils/` at least that way is not in the same place